### PR TITLE
Create posts working

### PR DESCRIPTION
**Pull request by FayeCarter**
Check out these posts

### CODE REVIEW

File: app/views/test.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<form method="post" action='http://localhost:3000/api/v1/posts'>
+  <input type="text" name="message">
+</form>
```
Comment (hturnbull93) [inline app/views/test.html]: We probably don't want this as it was for that one off manual test right?

File: spec/features/api_can_create_post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.feature "Posts API", type: :feature do
+  scenario "Visiting /api/v1/posts?message=Hello&slug=HelloWorldreturns all posts" do
+
+    page.driver.post('/api/v1/posts?message=Hello') 
+    expect(page).to have_content('"message":"Hello"')
+  end
+end
```
Comment (hturnbull93) [inline spec/features/api_can_create_post_spec.rb:9]: Great functional test :)

**Review by hturnbull93 — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/posts_controller.rb [hunk 3]
```diff
@@ -49,6 +49,6 @@ def set_post
   end
 
   def post_params
-    params.require(:post).permit(:message)
+    params.permit(:message)
```
Comment (tommyrharper) [inline app/controllers/api/v1/posts_controller.rb:52]: amazing

File: app/views/test.html (hunk 1 above)
Comment (tommyrharper) [inline app/views/test.html]: Also add an extra line

File: spec/features/api_can_create_post_spec.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.feature "Posts API", type: :feature do
+  scenario "Visiting /api/v1/posts?message=Hello&slug=HelloWorldreturns all posts" do
```
Comment (tommyrharper) [inline spec/features/api_can_create_post_spec.rb:4]: Good testing

**Pull request merged**